packages/ui/components/ui/testnet-banner/index.tsx

```diff
@@ -1,12 +1,13 @@
+import { Banner } from '../banner';
+
 export const TestnetBanner = ({ chainId }: { chainId?: string }) =>
```

Comment:
think we can remove `TestnetBanner`

Reply:
@grod220 confirm please

Reply:
Nah, let's keep this for now. Think it remains important to somehow distinguish visually mainnet vs testnet.